Assets/Scripts/Player/AttackBuffer.cs

```diff
@@ -88,5 +88,5 @@ public enum AttackDirection {
 public enum AttackType {
     NONE = 0,
     PUNCH = 1,
-    KICK = 2
+    KICK = 2,
```

Comment:
Nice

Reply:
Another idea that didn't really work out.